packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -594,6 +594,26 @@ void main() {
     FileSystem: () => MemoryFileSystem.test(),
     ProcessManager: () => FakeProcessManager.any(),
   });
+
+  testUsingContext('Does not support "--web-launch-url"', () async {
```

Comment:
"does not support" seems to be the opposite of what the test does?

Reply:
fixed